concordia/models.py

```diff
@@ -66,6 +68,41 @@ class Collection(models.Model):
     def __str__(self):
         return self.title
 
+    def copy_images_to_collection(self, url, collection_path):
+        result2 = None
+        try:
+            result = download_async_collection.delay(url)
+            result.ready()
+            result.get()
+            result2 = check_completeness.delay()
+            result2.ready()
+            result2.get()
+        except Exception as e:
+            print(e)
+            pass
+
+        if result2 and not result2.state == 'PENDING':
+            if os.path.isdir(collection_path):
+                shutil.rmtree(collection_path)
+            shutil.copytree('/concordia_images', collection_path)
+            for the_dir in os.listdir('/concordia_images'):
+                shutil.rmtree(os.path.join('/concordia_images', the_dir))
+
+    def create_assets_from_filesystem(self, collection_path):
+        for root, dirs, files in os.walk(collection_path):
+            for filename in files:
+                file_path = os.path.join(root, filename)
+                title = file_path.replace(collection_path + '/', '').split('/')[0]
```

Comment:
Use `os.path.join` instead of string concatenation. 
Use `os.path.dirname` instead of splitting the strings.

Does something else in the codebase prevent directories from being nested more than one level deep?